src/pages/instances/Events.tsx

```diff
@@ -47,8 +51,9 @@ const Events: FC = () => {
         void queryClient.invalidateQueries({
           queryKey: [queryKeys.operations, event.project],
         });
+        void refetchOperations();
       }
-      setTimeout(() => handleEvent(event), 1000); // handle with delay to allow operations to vanish
+      setTimeout(() => handleEvent(event), 3000); // handle with delay to allow operations to vanish
```

Comment:
Why increase this to 3 seconds? This makes LXD appear a lot less responsive.

Reply:
sorry I forgot to reverse this after experimenting. I found that the operations fetched from the server is not perfectly in sync with messages from the websocket so some delay is required or else we might have the number of operations stuck unless if we do a browser refresh.

Reply:
so I put the delay on event handling to 1500ms which makes the ui feel a lot more fluid, but increased the `refetchOperations` delay to 3500ms to allow for operations to dissipate consistently, this seems to work consistently as well.

Reply:
I think a better solution is to only consider "running" operations in operationStatus. Then we don't need to change the timing here and can remove the refetch timer in this `Events` component.